config/initializers/omniauth.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.config.middleware.use OmniAuth::Builder do
+  provider :developer unless Rails.env.production?
+  provider :google_oauth2, ENV['GOOGLE_KEY'], ENV['GOOGLE_SECRET']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.